content/en/docs/contributing/release-process.md

```diff
@@ -111,30 +154,66 @@ The process for cutting a minor release is as follows:
      the GitHub project to be able to push to the release branch.
 
      ```bash
+     # Must be run from the cert-manger repo folder.
      git push --set-upstream origin release-1.0
      ```
 
 4. Generate and edit the release notes:
 
-   1. Run as follows, substituting the current and last versions where appropriate:
+   1. Use the following two tables to understand how to fill in the 4
```

Comment:
Might be helpful to add a link to the k8s docs for the tool? https://github.com/kubernetes/release/blob/f69aaae60a036a7d33c120b3f0e8ad565646aa75/cmd/release-notes/README.md#options

Reply:
Thanks! Fixed in 2909939 👍

Reply:
Done in 8f9273fd6c45b3819f7563cf7245da5e02a504ae! thanks!

Reply:
Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba